builds/installing-openshift-builds.adoc

```diff
@@ -27,7 +27,7 @@ include::modules/ob-installing-builds-using-CLI.adoc[leveloffset=+1]
 include::modules/ob-creating-a-shipwright-build-resource-cli.adoc[leveloffset=+2]
 
 
+[role="_additional-resources"]
+== Additional resources
 
-// [role="_additional-resources"]
-// == Additional resources
-
+* You can learn more about installing Operators on {product-title} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-adding-operators-to-a-cluster[adding Operators to a cluster] section.
```

Comment:
@agantony @snarayan-redhat, I would try to set up the context a bit. The content is going to be merged in a new repo `build-docs` as we are proceeding to create a separate standalone branch for OpenShift Builds (one of the layered products) like we did for serverless a few months back. I cannot use xrefs because we do not have this module in our `build-docs` repos. The content is being created from scratch and we would keep only those modules that are part of OpenShift Builds.  The suggested workaround was to use links, rather using xref.

Reply:
got that! Thanks for clarifying @shipsing